homeassistant/components/broadlink/updater.py

```diff
@@ -161,6 +162,14 @@ async def async_fetch_data(self):
         return data
 
 
+class BroadlinkBG1UpdateManager(BroadlinkUpdateManager):
+    """Manages updates for Broadlink BG1 devices."""
+
+    async def async_fetch_data(self):
+        """Fetch data from the device."""
+        return await self.device.async_request(self.device.api.get_state)
+
```

Comment:
Remove the whitespace.

Reply:
which whitespace in particular?

Reply:
There is a whitespace at 171 causing a W293 error.

Reply:
hopefully that's fixed it. It was put in by the github editor when resolving the conflict :|